### ISSUE TITLE
Accessibility: MAS17: Color Contrast : Luminosity contrast ratio for the Iinks do not match 4.5:1 .

### ISSUE THREAD

**Issue by greazer** [Microsoft/vscode#29350]
Platform:
Microsoft Windows RS1
Version:10.0.14393.821
Microsoft Visual Studio Enterprise 2017 RC
Visual Studio/15.0.0-RC.4+26206.0

Steps to reproduce:
1. Launch Visual Studio Code.
2.Navigate to view menu.
3. Select "Extension" from view menu.
4. Select any of the extensions(example: select C# 1.8.1).
5.Observe the color contrast ratio for the links in the selected Extension tab using Contrast analyzer Application.

Expected Result:
Luminosity contrast ratio for the links should be 4.5:1.

Actual Result:
Luminosity contrast ratio for the links is 4.1:1.

User Impact:
People with low vision often have difficulty while reading the selected text that do not contrast with their background color.

![color contrast ratio for the links are minimum](https://user-images.githubusercontent.com/11685408/27495569-4560ebb2-5807-11e7-9104-0a06acff1230.png)
